src/components/crossForm/CrossForm.vue

```diff
@@ -66,7 +73,7 @@
               class="outline form-control text-center"
               :class="{ disabled: disabled }"
               placeholder="Amount"
-              :disabled="disabled"
+              :disabled="disabled || disabledAmount"
```

Comment:
not sure if disabled make sense at this point. If so change
```js
 :class="{ disabled: disabled }"
 ```
 To follow the same rule as :disabled

Reply:
it was under the v-if="false" so it was removed.